CoalesceSample.Data/Models/Game.cs

```diff
@@ -7,4 +7,6 @@ public class Game
     public double AverageDurationInHours { get; set; }
     public int MaxPlayers { get; set; }
     public int MinPlayers { get; set; }
+    public int GenreId { get; set; }
+    public ICollection<Tag> GameTags { get; set; } = new List<Tag>();
```

Comment:
Does this actually work being a Tag and not a GameTag? What mechanism populates this?